Dockerfile

```diff
@@ -86,36 +89,34 @@ WORKDIR /
 COPY --from=pgsql-stage /usr/lib/postgresql /usr/lib/postgresql
 COPY --from=pgsql-stage /usr/share/postgresql /usr/share/postgresql
 
-## Copy from dashboard-stage
-WORKDIR /dashboard
-COPY --from=dashboard-stage /dashboard/*.* ./
-COPY --from=dashboard-stage /dashboard/src src
-COPY --from=dashboard-stage /dashboard/public public
-COPY --from=dashboard-stage /dashboard/node_modules node_modules
+## Copy files
+COPY ./src/scripts/postgresql.conf /etc/postgresql/14/main/postgresql.conf
+COPY ./src/scripts/pg_hba.conf /etc/postgresql/14/main/pg_hba.conf
+COPY docker-entrypoint.sh /engine/docker-entrypoint.sh
+COPY src/scripts /engine/src/scripts
 
 ## Copy from engine-stage
 WORKDIR /engine
-COPY --from=engine-stage /engine/package.json ./
 COPY --from=engine-stage /engine/node_modules node_modules
+COPY --from=engine-stage /engine/package.json ./
```

Comment:
Why swap the order on these two lines?

Reply:
`node_modules` layer should be defined as early as we can in the Dockerfile to take advantage of the cache. The engine's `niode_modules` take a considerable time for a small code change.